### DISCUSSION TITLE
Can Crazyflie use Lighthouse and Loco Positioning together?

### DISCUSSION THREAD

**Discussion by armorbear00** [bitcraze/discussions#1675] [General]
I’ve been asked whether it is possible to use the Lighthouse system and the Loco Positioning system simultaneously with Crazyflie. If it is not possible, could you clarify whether the limitation is due to hardware constraints, software design, or both? I would greatly appreciate your response to this question.

**Comment by gemenerik** [comment]
As you can find in [the deck-deck compatibility matrix](https://www.bitcraze.io/documentation/system/platform/cf2-expansiondecks/#deck---deck); the Lighthouse and Loco positioning system do not work together out of the box. [They share some pins](https://www.bitcraze.io/documentation/system/platform/cf2-expansiondecks/#deck-pin-allocation). As you can see in the compatibility matrix, you could make them compatible with a hardware patch (using soldering bridges) + a change to the firmware. The hardware mod is not trivial. [This blogpost about the "hyper demo"](https://www.bitcraze.io/2020/12/the-hyper-demo/), where we implemented these modifications ourselves, can get you started.
